exotica_core/src/problems/end_pose_problem.cpp

```diff
@@ -30,9 +30,9 @@
  *
  */
 
-#include <exotica/Problems/EndPoseProblem.h>
-#include <exotica/Setup.h>
-#include <exotica/TaskInitializer.h>
+#include <exotica_core/TaskInitializer.h>
```

Comment:
I always split off groups of includes by an empty line:
1. System headers (C first, then C++)
2. exotica_core includes
3. Package or other dependency includes
4. Initializers